tests/FSharp.Test.Utilities/Utilities.fs

```diff
@@ -69,20 +54,6 @@ module Utilities =
                 cancellationToken)
             task.Result
 
-    /// Disposable type to implement a simple resolve handler that searches the currently loaded assemblies to see if the requested assembly is already loaded.
-    type AlreadyLoadedAppDomainResolver () =
-        let resolveHandler =
```

Comment:
Now this is a hard question because it was some time ago :)
IIRC this handler is installed on the top-level app domain and basically does nothing, because if the domain have an assembly loaded, it won't invoke resolvers. I think at some point I put a `failwith` in there to confirm my suspicion this code is defunct. I'll recheck because I might remember wrong.